modulith/build.gradle

```diff
@@ -1,40 +1,32 @@
 plugins {
     id 'java'
-    id 'org.springframework.boot' version '3.1.0' apply false
+    id 'idea'
     id 'io.spring.dependency-management' version '1.1.0' apply false
+    id 'org.javamodularity.moduleplugin' version '1.8.12' apply false
 }
 
 repositories {
     mavenCentral()
 }
 
-dependencies {
-}
-
 allprojects {
     group = 'io.edpn.backend'
     version = '0.0.1-SNAPSHOT'
     compileJava.options.encoding = 'UTF-8'
 }
 
 subprojects {
-    apply plugin: 'java'
-    apply plugin: 'org.springframework.boot'
-    apply plugin: 'io.spring.dependency-management'
-
-    sourceCompatibility = JavaVersion.VERSION_17
-    targetCompatibility = JavaVersion.VERSION_17
+    apply plugin: "java"
+    apply plugin: "idea"
+    apply plugin: "org.javamodularity.moduleplugin"
 
+    version = '0.0.1-SNAPSHOT'
 
     repositories {
         mavenCentral()
     }
-
-    dependencies {
-        implementation 'de.slub-dresden:urnlib:2.0.1'
-
-        compileOnly 'org.projectlombok:lombok:1.18.28'
-    }
+    task prepareKotlinBuildScriptModel {}
```

Comment:
why add kotlin tasks

Reply:
It was the fix suggested online. the wrapper and prepareKotlin tasks are complained about to be missing on the subprojects by gradle. not sure what the cause is, but the fix works for now